HW03/maksim_manzhai/maksim_manzhai_hw_03_t_03.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+require 'time'
+
+class Homework3
+  def task3(log)
+    lines = log.split(/\n/)
+
+    lines.map! { |line| Time.parse(line) if event_valid?(line) }
```

Comment:
It seems to me that it is better to select all the matching lines and manipulate them, rather than checking the match every time through "if"